### PR TITLE
docs: Remove onramp kit from onramp guides

### PR DESCRIPTION
**Pull request by valle-xyz**

### CODE REVIEW

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 1]
```diff
@@ -1,119 +1,152 @@
-import { Steps } from 'nextra/components'
+import { Steps, Callout } from 'nextra/components'
+import Image from 'next/image'
 
 # Integration with Stripe
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:5]: Can we add some information about what is possible through this Integration like you did for Monerium here? https://github.com/safe-global/safe-docs/pull/553/files#diff-efc102eaaa40c277c1a243efd02c54933ba1f39eb22ab662d1cac4add1b6f276R7

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 2]
```diff
@@ -1,119 +1,152 @@
...
 
-The [Stripe Crypto Onramp service](https://stripe.com/docs/crypto/overview) allows individuals to securely purchase cryptocurrencies from your application. 
+<Callout type="info" emoji="ℹ️">
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:6]: @germartinez might have thoughts on if we should remove this as he was working on a different style on redirection. Might be good to check this with him.

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 3]
```diff
@@ -1,119 +1,152 @@
...
-    // }
+### Obtain your public and private keys
+
+To use the Stripe Crypto Onramp service, you need to obtain your [public and private keys](https://docs.stripe.com/keys).
+You have to apply for the crypto onramp service and add at least your business address and information to your Stripe account.
+When your application is approved, you will find your public and private keys in your [Stripe Developer Dashboard](https://dashboard.stripe.com/test/apikeys).
+
+### Install the SDK and client library
+
+You can install the Stripe SDK as a script or as a package.
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:22]:
```suggestion
You can add the Stripe SDK as a script or install it as an npm package. 
```

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 4]
```diff
@@ -1,119 +1,152 @@
...
+When your application is approved, you will find your public and private keys in your [Stripe Developer Dashboard](https://dashboard.stripe.com/test/apikeys).
+
+### Install the SDK and client library
+
+You can install the Stripe SDK as a script or as a package. 
+
+<CH.Section>
+  <CH.Code style={{boxShadow: 'none'}}>
+
+    ```html script
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:27]:
```suggestion
    ```html Script
```

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 5]
```diff
@@ -1,119 +1,152 @@
...
+    <head>
+      <title>Onramp</title>
+      <script src="https://js.stripe.com/v3/"></script>
+      <script src="https://crypto-js.stripe.com/crypto-onramp-outer.js"></script>
+    </head>
+    ```
+
+    ```bash package
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:35]:
```suggestion
    ```bash npm
```

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 6]
```diff
@@ -1,119 +1,152 @@
...
+    npm install --save @stripe/stripe-js @stripe/crypto
+    ```
+  </CH.Code>
+</CH.Section>
+
+### Generate a crypto onramp session
+
+You need to generate a crypto onramp session using your stripe private key.
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:43]: Should these be numbered steps (if they should be done in order)?

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 7]
```diff
@@ -1,119 +1,152 @@
...
+
+As you use your private key, the request must be made from your backend.
+The backend can then send the `client_secret` to your frontend.
+You can use the [Safe example Stripe](https://github.com/safe-global/safe-core-sdk/tree/main/packages/onramp-kit/example/server) server as a starting point for your backend.
+
+Here is how you generate a crypto onramp session using your private key:
+
+<CH.Section>
+  <CH.Code style={{boxShadow: 'none'}}>
+    ```typescript typescript
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:55]:
```suggestion
    ```typescript TypeScript
```

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 8]
```diff
@@ -1,119 +1,152 @@
...
+        -u sk_test_51...Eg7o: \
+        # optional parameters, for example a wallet address
+        -d "wallet_addresses[ethereum]"="0xB00F0759DbeeF5E543Cc3E3B07A6442F5f3928a2"
+    ```
+  </CH.Code>
+</CH.Section>
+
+### Initialize the Stripe widget
+
+The stripe widget is a secure iframe that allows users to purchase cryptocurrencies.
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:85]:
```suggestion
The Stripe widget is a secure iframe that allows users to purchase cryptocurrencies.
```

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 9]
```diff
@@ -1,119 +1,152 @@
...
-  ...
+![The Stripe widget](https://b.stripecdn.com/docs-statics-srv/assets/crypto-onramp-overview.c4c0682697f2cd4c1c2769c3c5e08506.png)
 
-  stripePack.unsubscribe('onramp_ui_loaded', uiLoadedHandler)
-  stripePack.unsubscribe('onramp_session_updated', sessionUpdatedHandler)
-  ```
+## Test the Stripe widget
 
-  ### Test the Stripe widget
+  In production, each customer should pass an individual KYC process but probably you want to test your application before 😊.
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:142]:
```suggestion
  In production, each customer should pass an individual KYC process but probably you want to test your application before. 
```

File: pages/sdk/onramp-kit/guides/stripe.mdx [hunk 10]
```diff
@@ -129,18 +162,26 @@ We are going to learn how to render the Stripe widget into your page. This widge
...
 
-![Payment Method](https://user-images.githubusercontent.com/9806858/228418059-b83b6357-a6b0-4f09-a4b2-3b89767cb4f0.png)
+## Conclusion
 
-## StripePack complete React example
+Well done, you have successfully integrated the Stripe Crypto Onramp service into your application.
+Your users can now purchase cryptocurrencies securely within your app.
 
-Check a complete [example](https://github.com/safe-global/safe-core-sdk/blob/main/packages/onramp-kit/example/client) in the `safe-core-sdk` repository. Follow the steps in the [`README.md`](https://github.com/safe-global/safe-core-sdk/blob/main/packages/onramp-kit/example/client/README.md) to run the example and configure the environment variables (`VITE_MONERIUM_CLIENT_ID` isn't necessary) for the pack following the [`.env.sample`](https://github.com/safe-global/safe-core-sdk/blob/main/packages/onramp-kit/example/client/.env.sample).
+If you have any questions or encounter any issues, the [Stripe support](https://support.stripe.com/) is responsive and helpful.
```
Comment (tanay1337) [inline pages/sdk/onramp-kit/guides/stripe.mdx:187]:
```suggestion
If you have any questions or encounter any issues, reach out to the [Stripe support](https://support.stripe.com/) team.
```

**Review by tanay1337 — CHANGES_REQUESTED** [review]
A few general notes:

- Please run the content through Grammarly Pro.
- You can delete the references and guides that these new pages replace in this PR.
- You can move them to a new location (let's discuss this).

File: pages/sdk/onramp-kit/guides/stripe.mdx (hunk 2 above)
Comment (valle-xyz) [reply pages/sdk/onramp-kit/guides/stripe.mdx:6]: @germartinez I will wait for our discussion to resolve this comment. I see three possibilities: 1. I remove the callout. 2. I make it optional. 3. I leave it as it is.

I am also happy to create an importable component that checks for a get parameter in the URL and displays a Callout conditionally.

File: pages/sdk/onramp-kit/guides/stripe.mdx (hunk 6 above)
Comment (valle-xyz) [reply pages/sdk/onramp-kit/guides/stripe.mdx:43]: Hey, not sure which steps you mean here.

Comment (valle-xyz) [reply pages/sdk/onramp-kit/guides/stripe.mdx]: The steps container creates numbered steps from the "###" headlines. Did you mean this?

**Comment by valle-xyz** [comment]
There are two open comments:

1. To make the Callout at the top of each guide conditional.
2. To add numbered steps.

Despite these comments, the PR is ready for review.

File: pages/sdk/onramp-kit/guides/stripe.mdx (hunk 2 above)
Comment (valle-xyz) [reply pages/sdk/onramp-kit/guides/stripe.mdx]: Resolved: We leave it like it is for this PR and create a new PR to make it conditional.

**Review by valle-xyz — COMMENTED** [review]
thanks

File: pages/sdk/onramp/monerium.mdx [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+Now that the user is authenticated with Monerium, you can place an order to transfer tokens from the Safe to an IBAN.
+
+<Steps>
+
+### Get some tokens
+
+First, your user needs to obtain some EURe test tokens on Sepolia:
+
+* Log into the [Monerium Sandbox](https://sandbox.monerium.dev/accounts) account.
+* Click on the `Add Money` button.
```
Comment (tanay1337) [inline pages/sdk/onramp/monerium.mdx:157]: The button in the attached screenshot says "ADD MONEY".

**Review by tanay1337 — CHANGES_REQUESTED** [review]

File: pages/sdk/onramp.mdx [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import { Grid } from '@mui/material'
+import CustomCard from '../../components/CustomCard'
+
+# Onramp guides
```
[the remark this answers is not in the document]
Comment (valle-xyz) [reply pages/sdk/onramp.mdx:4]: Good idea. I just realized that I have to create an example repo for that, which is generally a good idea.

File: pages/sdk/onramp/monerium.mdx [hunk 13]
```diff
@@ -0,0 +1,245 @@
+import { Steps } from 'nextra/components'
+import Image from 'next/image'
+import MoneriumAddMoney from '../../../assets/monerium-sandbox-add-money.png'
+
+# Onramping with Monerium
+
+The [Monerium SDK](https://www.npmjs.com/package/@monerium/sdk) enables Safe users to directly transfer e-money tokens from their Safe accounts to an IBAN via the SEPA network.
+
+More info about Monerium:
```
[the remark this answers is not in the document]
Comment (valle-xyz) [reply pages/sdk/onramp/monerium.mdx:9]: Yes! Should I remove the link in the paragraph, or is it good to have a duplicated link?

[the remark this answers is not in the document]
Comment (tanay1337) [reply pages/sdk/onramp/monerium.mdx:9]: Best to not have duplicates to avoid content debt.

File: pages/sdk/onramp/monerium.mdx [hunk 14]
```diff
@@ -0,0 +1,249 @@
...
+    }
+  },
+  message: orderMessage,
+  memo: 'Powered by Monerium SDK',
+  chain: 'ethereum',
+  network: 'sepolia'
+})
+```
+
+Monerium will listen for the sign message transaction of the following step on the chain and execute the order once the transaction settles.
```
Comment (germartinez) [inline pages/sdk/onramp/monerium.mdx:200]:
```suggestion
Monerium will listen for the sign message transaction of the following step on the selected chain and execute the order once the transaction settles.
```

File: pages/sdk/onramp/monerium.mdx [hunk 15]
```diff
@@ -0,0 +1,249 @@
...
+* Log into the [Monerium Sandbox](https://sandbox.monerium.dev/accounts) account.
+* Click on the `ADD MONEY` button.
+* Create a test IBAN transfer onto the account. The tokens from this test transfer will be available in the Safe.
+
+<Image src={MoneriumAddMoney} width={300} height={500}  />
+
+### Send an order to Monerium
+
+To send tokens from the Safe to an IBAN, the user must sign a message with the Safe and send the order to Monerium. 
+You can trigger both actions simultaneously.
```
Comment (germartinez) [inline pages/sdk/onramp/monerium.mdx:165]: I would rephrase it this way for simplicity and coherence with the order of our steps:
```suggestion
To send tokens from a Safe to an IBAN, users must send the order to Monerium and sign a message with their account.
```

File: pages/sdk/onramp/monerium.mdx [hunk 16]
```diff
@@ -0,0 +1,249 @@
...
+  address: safeAddress, // the Safe address
+  counterpart: {
+    identifier: {
+      standard: 'iban',
+      iban
+    },
+    details: {
+      firstName: 'User',
+      lastName: 'Userson',
+      county: 'AL'
```
[the remark this answers is not in the document]
Comment (valle-xyz) [reply pages/sdk/onramp/monerium.mdx:190]: Thanks, it actually is County. 

grafik https://www.npmjs.com/package/@monerium/sdk

File: pages/sdk/onramp.mdx [hunk 17]
```diff
@@ -0,0 +1,13 @@
...
+
+The guides in this section show how to use different onramp services with the Safe\{Core\} SDK. 
+With these onramp services, you can enable users to buy cryptocurrencies using their credit card or bank account.
+With Monerium, you can even send IBAN transfers from a Safe account to a bank account.
+
+The following guides show how to do onramping with the following services:
+
+- [Monerium](./onramp/monerium.mdx)
```
Comment (tanay1337) [inline pages/sdk/onramp.mdx:12]: The listing sequence here is different than in the sidebar. Can we please make it consistent? Would prefer alphabetical ordering (since there's no prio or sequence).

File: pages/sdk/onramp/monerium.mdx (hunk 13 above)
[the remark this answers is not in the document]
Comment (tanay1337) [reply pages/sdk/onramp/monerium.mdx:9]: This was not resolved, we should link the mentions in the paragraph above instead of dangling links.

File: pages/sdk/onramp/stripe.mdx [hunk 18]
```diff
@@ -0,0 +1,203 @@
+import { Steps, Callout } from 'nextra/components'
+import Image from 'next/image'
+import StripeKycPersonalInfoImage from '../../../assets/stripe-kyc-personal-info.png'
+import KycAddress from '../../../assets/stripe-kyc-address.png'
+import PaymentMethod from '../../../assets/stripe-kyc-payment-method.png'
+
+# Onramp with Stripe
+
+<Callout type="info" emoji="ℹ️">
```
Comment (tanay1337) [inline pages/sdk/onramp/stripe.mdx:9]: Should this be removed as Germán has another PR open for this?

File: pages/sdk/onramp/stripe.mdx [hunk 19]
```diff
@@ -0,0 +1,203 @@
...
+  With the Onramp SDK deprecated, developers are now directed to integrate directly with Stripe’s Crypto Onramp service.
+</Callout>
+
+The Stripe fiat-to-crypto onramp service allows you to integrate a secure widget into your application that enables users to purchase cryptocurrencies using their credit card or bank account.
```
Comment (tanay1337) [inline pages/sdk/onramp/stripe.mdx:13]: Can you please link to the Stripe fiat-to-crypto onramp service?

**Review by tanay1337 — CHANGES_REQUESTED** [review]

File: pages/sdk/onramp/monerium.mdx [hunk 20]
```diff
@@ -0,0 +1,249 @@
...
+// Create a transaction
+const signMessageTx = await protocolKit.createTransaction({
+  transactions: [safeTransactionData]
+})
+
+// Sign the transaction with the first owner
+const signedTx = await protocolKit.signTransaction(signMessageTx)
+
+// Connect the protocol kit to the second owner
+const protocolKitOfOwner2 = await protocolKit.connect({
```
[the remark this answers is not in the document]
Comment (valle-xyz) [reply pages/sdk/onramp/monerium.mdx:87]: That is a very good point! I am on the brink. On one hand, you are right, and the confirmation happens on the second owner's device. On the other hand, this makes the code more complex, and the focus of this tutorial is the onramping with Monerium. I am like 51% leaning towards leaving this as it is, but I am open to changing it.

File: pages/sdk/onramp/monerium.mdx (hunk 13 above)
[the remark this answers is not in the document]
Comment (valle-xyz) [reply pages/sdk/onramp/monerium.mdx:9]: Ok, now I am a little confused. What do you prefer? To add it to a 'further reading' section, or to leave it in the paragraph?

File: pages/sdk/onramp/monerium.mdx [hunk 21]
```diff
@@ -0,0 +1,249 @@
...
+```
+
+Calling `authorize` will redirect the user to the Monerium login page. 
+
+### Authenticate with Monerium
+
+At Monerium, the user needs to log in or create an account. 
+Once logged in, Monerium will verify the ownership of the Safe by checking the signed message. 
+After successful verification, Monerium will create an IBAN and link it to the Safe. 
+The user will then be redirected to the specified `redirectUrl` with an added `code` query parameter.
```
[the remark this answers is not in the document]
Comment (valle-xyz) [reply pages/sdk/onramp/monerium.mdx]: I replaced the text with "The users will then be redirected to the specified `redirectUrl` with the new session id as a GET parameter." wdyt?

File: pages/sdk/onramp/stripe.mdx (hunk 18 above)
Comment (valle-xyz) [reply pages/sdk/onramp/stripe.mdx:9]: I will remove it as we can always add it back.

File: pages/sdk/onramp/monerium.mdx (hunk 21 above)
[the remark this answers is not in the document]
Comment (germartinez) [reply pages/sdk/onramp/monerium.mdx]: Sounds good!

File: pages/sdk/onramp/monerium.mdx (hunk 20 above)
[the remark this answers is not in the document]
Comment (germartinez) [reply pages/sdk/onramp/monerium.mdx:78]: OKay, let's leave it as it is.

File: pages/sdk/onramp/stripe.mdx [hunk 22]
```diff
@@ -0,0 +1,205 @@
...
+
+<Steps>
+
+### Obtain your public and private keys
+
+To use the Stripe fiat-to-crypto onramp service, you need to obtain your [public and private keys](https://docs.stripe.com/keys).
+You have to apply for the crypto onramp service and add at least your business address and information to your Stripe account.
+When your application is approved, you will find your public and private keys in your [Stripe Developer Dashboard](https://dashboard.stripe.com/test/apikeys).
+
+### Install the client library
```
[the remark this answers is not in the document]
Comment (germartinez) [reply pages/sdk/onramp/stripe.mdx]: Do you mean that you would prefer having all the code blocks inside the `Integrate the Stripe fiat-to-crypto onramp widget` section? Sounds good to me.

However, imports are still missing. Having them in a separate section or the existing code snippets it's fine for me.

[the remark this answers is not in the document]
Comment (germartinez) [reply pages/sdk/onramp/stripe.mdx]: I see this comment but I can't find it
```
// StripeOnramp is imported with the scripts from step one
```

**Review by germartinez — APPROVED** [review]
